src/components/CodeBox.vue

```diff
@@ -60,6 +66,14 @@ function isWordSelectedBug(wordId: number) {
   return selectedBugs.value.find((bug) => bug.wordId == wordId) != null;
 }
 
+function isWordSpace(wordId: number): boolean {
```

Comment:
Perhaps this function should be enhanced to `isWordWhitespace`, so that `Tabs` will be handled correctly as well?

Reply:
Tabs should never be interactive. So there is no added value to it.

Reply:
> Tabs should never be interactive

I disagree. As the user can supply his own code samples, you can't predict what he wants to do.
It is a valid case that he wants to have a bug at a place where a tab is, so that should be handled just the same.

Reply:
I support the opinion of @maexled. Tabs shouldn't be interactive. I only see a small additional value but a not negligible disadvantage. This would lead to a bad user experience because the user didn't knows where he has to click to insert a word in front of an actual line. If he accidentally clicks the tab instead the space he doesn't get the point although he recognized the bug corrwctly.  Also I don't know how we should handle tab-errors. How can they be corrected? The only problem why we should handle tabs is python. Python intendation-errors couldn't be handled in our bugfinder.

Reply:
What we could do:
If there is more than one subsequent occurrence of whitespace, it is only counted as one.
But that's definitely not in scope for this PR.